cachito/web/api_v1.py

```diff
@@ -31,9 +31,32 @@
 from cachito.web.utils import deep_sort_icm, pagination_metadata, str_to_bool
 from cachito.workers import tasks
 
+from prometheus_client import multiprocess
+from prometheus_client.core import CollectorRegistry
+from prometheus_flask_exporter import PrometheusMetrics
+
+
 api_v1 = flask.Blueprint("api_v1", __name__)
 
 
+def init_metrics():
+    """
+    Initialize the Prometheus Flask Exporter.
+
+    :param none
+    :return: a Prometheus Flash Metrics object
+    :rtype: PrometheusMetrics.for_app_factory
+    """
+    registry = CollectorRegistry()
+    if not os.path.isdir(os.environ["PROMETHEUS_MULTIPROC_DIR"]):
+        os.mkdir(os.environ["PROMETHEUS_MULTIPROC_DIR"])
```

Comment:
https://docs.python.org/3/library/os.html#os.makedirs  you porbably want to use this, due nested dirs

Reply:
Good idea!